examples/meta/generator/parse.py

```diff
@@ -72,6 +72,7 @@ def addShogunTypes(self, dictionary):
         'int': 'BASICTYPE',
         'bool': 'BASICTYPE',
         'float':  'BASICTYPE',
+        'real': 'BASICTYPE',
```

Comment:
you should rebase this branch against develop to not clutter the diff with these things. Makes it hard to read